tests/psalm-baseline.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="4.7.3@38c452ae584467e939d55377aaf83b5a26f19dd1">
+  <file src="lib/BackgroundJob.php">
+    <NullArgument occurrences="1">
+      <code>null</code>
+    </NullArgument>
+  </file>
+  <file src="lib/Dashboard/Widget.php">
+    <InvalidArgument occurrences="1"/>
+  </file>
+  <file src="lib/Model/Announcement.php">
+    <UndefinedMethod occurrences="2">
+      <code>parent::getMessage()</code>
```

Comment:
why is it not `$this->getMessage()`? I think Psalm is right here.

Reply:
For message this works, or is the same. On getSubject() I didn't use a different name in my model.
And the problem is that `OCP\AppFramework\Db\Entity` works with `__call` and therefore yes Psalm is right, the method does not exist, but it works :-X

Reply:
I see. Carry on :speak_no_evil:

Reply:
Fixed